charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -98,6 +98,9 @@ spec:
           {{- with .Values.kubeRBACProxy.resources }}
           resources: {{ toYaml . | nindent 12 }}
           {{- end }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets: {{ toYaml .Values.imagePullSecrets | nindent 8 }}
```

Comment:
This looks a little different than the collector chart's implementation.  Is that intentional?

Reply:
Shouldn't it be nested?

Reply:
Haven't looked at the collector implementation before. Changed it to match and tested.